combine.lua

```diff
@@ -0,0 +1,496 @@
+-- Combines food and plant items across stockpiles.
+local argparse = require('argparse')
+local utils = require('utils')
+
+-- used when the info option is specified
+local out_fp = nil
+local out_name = ''
+local out_err = nil
+
+local opts, args = {
+    help = false,
+    preview = false,
+    merge = false,
+    stockpile = nil,
+    types = nil,
+    max = 0,
+    info = false,
+    debug = 0
+  }, {...}
+
+local valid_mode_list = {
+    'preview',
+    'merge'
+}
+
+local valid_modes = utils.invert(valid_mode_list)
+
+local valid_types_list = {
+    'all',
+    'drink',
+    'fat',
+    'fish',
+    'meal',
+    'meat',
+    'plant'
+}
+
+local valid_types = utils.invert(valid_types_list)
```

Comment:
these two can be removed. the single reference to `valid_types` can instead use `valid_types_map`.

Reply:
Yes, I can take them out. They were from the initial build.

Reply:
done.